vcd/resource_vcd_vapp_vm.go

```diff
@@ -33,6 +33,10 @@ func resourceVcdVAppVm() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"computer_name": &schema.Schema{
```

Comment:
Is this the host name of the VM?

Reply:
yes
![image](https://user-images.githubusercontent.com/41415459/65326923-42f17b00-dbbb-11e9-9522-9dd58a5c1fa9.png)